### PR TITLE
Containers Added

### PR DESCRIPTION
**Pull request by ankushg07**

### CODE REVIEW

File: c++.html.markdown [hunk 1]
```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
 // explanation of why this works.
 v.swap(vector<Foo>());
 
+
+/////////////////////
+// CONTAINERS
+/////////////////////
+
+//Containers or the Standard Template Library are some predefined templates
```
Comment (vendethiel) [inline c++.html.markdown]: please add a space between the `//` and the text, like the rest of the tutorial does

File: c++.html.markdown [hunk 2]
```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+//They  manages the storage space for its elements and provide
+//member functions to access and manipulate them
+
+//Few containers are as follows:-
+
+//Vectors (Dynamic array)
```
Comment (vendethiel) [inline c++.html.markdown]: arrays

File: c++.html.markdown [hunk 3]
```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+//Allow us to Define the Array or list of objects at run time
```
Comment (vendethiel) [inline c++.html.markdown]: no, that's not what it does

File: c++.html.markdown [hunk 4]
```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+Vector_name.push_back(val); // will push the value of variable into array
+
+//To iterate through vector, we have 2 choices
+//using normal looping 
+for(int i=0; i<Vector_name.size(); i++)
+//it will iterate through the vector from index '0' till last index
+
+// Using Iterator
+vector<Data_Type>::iterator it; //initialize the iteartor for vector
+for(it=vector_name.begin(); it!=vector_name.end();++it)
```
Comment (vendethiel) [inline c++.html.markdown]: what about C++11 for-range?

File: c++.html.markdown [hunk 5]
```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+//Containers or the Standard Template Library are some predefined templates
+//They  manages the storage space for its elements and provide
+//member functions to access and manipulate them
+
+//Few containers are as follows:-
+
+//Vectors (Dynamic array)
+//Allow us to Define the Array or list of objects at run time
+#include<vector>//will include the header file for vector
+vector< Data_Type > Vector_name; // used to initialize the vector
```
Comment (vendethiel) [inline c++.html.markdown]: please follow naming convention used in the file

File: c++.html.markdown [hunk 6]
```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
...
+ 10 20 30
+ 
+//To erase an element
+ST.erase(20); // will erase element with value 20
+//Set ST: 10 30
+//To iterate through Set we use iterators
+set< int >::iterator it;
+for(it=ST.begin();it<ST.end();it++)
+{
+	cout<<*it<<endl;
```
Comment (vendethiel) [inline c++.html.markdown]: please space operators and brackets (like `a<foo>`) like the rest of the file does

**Comment by vendethiel** [comment]
@ankushg07 any news?:)

**Comment by ankushg07** [comment]
@vendethiel I am sorry sir this was completely out of my mind will surely fix issues ASAP

**Comment by vendethiel** [comment]
No problem! Thank you very much

**Comment by ankushg07** [comment]
@vendethiel  sir many issues were not resolved but it got merged 
Sorry for disappointing you

**Comment by vendethiel** [comment]
it's fine :)

**Pull request merged**